tests/views/test_blacklist.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
I can think of a couple of more test cases:

* The superuser tries to blacklist the token
* `rest_framework_jwt.blacklist` is not registered in the `INSTALLED_APPS`
* User A tries to blacklist user B
* A user tries to blacklist a token that is already blacklisted.

Reply:
All suggested test cases, except the second bullet (**blacklist** app not registered in `INSTALLED_APPS`), have been added.